test/result-list-spec.js

```diff
@@ -1,38 +1,40 @@
+/* jshint esnext: true */
+
 'use strict';
 
-var chai = require('chai');
-var chaiAsPromised = require('chai-as-promised');
-var mockagent = require('mockagent');
-var pixabay = require('../src/index');
-var promiseHelpers = require('promisehelpers');
-var range = require('lodash.range');
-var ResultList = require('../src/result-list');
-var sinon = require('sinon');
-var sinonChai = require('sinon-chai');
-var superagent = require('superagent');
-var url = require('url');
-
-var notify = promiseHelpers.notify;
-var wrap = promiseHelpers.wrap;
-
-var expect = chai.expect;
+let chai = require('chai');
+let chaiAsPromised = require('chai-as-promised');
+let mockagent = require('mockagent');
+let pixabay = require('../src/index');
+let promiseHelpers = require('promisehelpers');
+let range = require('lodash.range');
+let ResultList = require('../src/result-list');
+let sinon = require('sinon');
+let sinonChai = require('sinon-chai');
+let superagent = require('superagent');
+let url = require('url');
+
+let notify = promiseHelpers.notify;
+let wrap = promiseHelpers.wrap;
+
+let expect = chai.expect;
```

Comment:
Not sure what I like more, desctructure here, or normal assign object dot value.